src/ZipFile.jl

```diff
@@ -487,6 +500,9 @@ end
 
 # Write nb elements located at p into f.
 function write(f::WritableFile, p::Ptr, nb::Integer)
+    if nb == 0
+        return 0
+    end
```

Comment:
I don't understand what's the bug. If nb is 0, `write` function wouldn't write anything anyway.